### PR TITLE
Adding disclaimer for app usage report

### PR DESCRIPTION
**Pull request by ivlists**
Adding note in app usage report that metrics are calculated only at channel level

### CODE REVIEW

**Comment by ivlists** [comment]
@cichur can you please help us get this pull request merged?

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/teams-analytics-and-reports/app-usage-report.md [hunk 1]
```diff
@@ -42,7 +42,9 @@ The Teams app usage report in the Microsoft Teams admin center provides you with
...
+|**6**   |The table gives you a breakdown of active users and teams by App name.<br><ul><li>**App name** is the display name of the app used in Teams.</li><li>**Active users** is the number of users who opened the app at least once during the specified time period.</li><li>**App type** is a static value of either “Microsoft” or “Third Party”.</li><li>**Active teams** is the number of teams who have opened the App by at least one member of the team and during the specified time periods.</li><li>**Publisher** is the software publisher of the app.</li><li>**Version** is the software version of the app, from the app publisher.</li></ul><b> Note :</b> Currently, 'Active users' and 'Active teams' are calculated for apps used in channels only.
```
Comment (JohanFreelancer9) [inline Teams/teams-analytics-and-reports/app-usage-report.md:45]:
```suggestion
|**6**   |The table gives you a breakdown of active users and teams by App name.<br><ul><li>**App name** is the display name of the app used in Teams.</li><li>**Active users** is the number of users who opened the app at least once during the specified time period.</li><li>**App type** is a static value of either “Microsoft” or “Third Party”.</li><li>**Active teams** is the number of teams who have had the app opened by at least one member of the team during the specified time period.</li><li>**Publisher** is the software publisher of the app.</li><li>**Version** is the software version of the app, from the app publisher.</li></ul><b> Note:</b> Currently, 'Active users' and 'Active teams' are calculated for apps used in channels only.     
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by ivlists** [comment]
@SerdarSoysal  Can we please get this PR merged? TIA.

**Pull request merged**